### PR TITLE
add tcp flags in iptables chain && minor fix

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
### What problem does this PR solve?

1. add tcp flags in iptables chain, used for network partition in chaosd https://github.com/chaos-mesh/chaosd/pull/65
2. some minor fix

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Breaking backward compatibility

Related changes

- [ ] Need to update the documentation

### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: pkg/chaosdaemon/tc_server.go [hunk 1]
```diff
@@ -214,23 +214,9 @@ func (s *DaemonServer) setFilterTcs(
...
-				ch.SourcePorts = fmt.Sprintf("-m multiport --source-ports %s", tc.SourcePort)
-			}
-		}
-
-		if len(tc.EgressPort) > 0 {
-			ch.DestinationPorts = fmt.Sprintf("--destination-port %s", tc.EgressPort)
-			if strings.Contains(tc.EgressPort, ",") {
-				ch.DestinationPorts = fmt.Sprintf("-m multiport --destination-ports %s", tc.EgressPort)
-			}
-		}
```
Comment (WangXiangUSTC) [inline pkg/chaosdaemon/tc_server.go:233]: I think it is better to keep these variables in the `Chain` clean, and generate these configs in iptables

**Review by AsterNighT — APPROVED** [review]

**Pull request merged**